_people/Henry_Huang.markdown

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Can you also add a headshot in `images/people/Henry_Huang.jpg`

Reply:
Done @nilsdeppe